webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -1358,8 +1580,6 @@
                 rdfs:label  "Manuskripteintrag in"@de ,
                             "Manuscript entry in"@en ;
 
-                knora-base:subjectClassConstraint :manuscriptEntry ;
```

Comment:
Why did you remove the subject class constraint?

Reply:
because this property is being used by the biblio:VPArticle resource as well.